push-notifications/basics/js.md

```diff
@@ -33,14 +33,14 @@ you by requesting permission to obtain push notification token.
 
 When the device obtained the push notification token, the `register`
 event is fired. Add an event listener for this event and use the
-`registerDevice` function to register the token with Skygear.
+[`registerDevice`][api-register-device] function to register the token with Skygear.
 
 ```js
 var { PushNotificationIOS } = require('react-native');
 var skygear = require('skygear');
 
 PushNotificationIOS.addEventListener('register', function(token) {
-  skygear.registerDevice(arg);
+  skygear.push.registerDevice(token, 'ios', 'changeme-topic');
```

Comment:
`changeme-topic` -> `bundleID/apckagename`

I don't think we want to introduce the idea of Topic in this guide, tell them it is `bundleID/packagename`.

If we introduce Topic, better to include the respective APNS and FMC link here. And explains a bit how is it works.

https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/CommunicatingwithAPNs.html
https://firebase.google.com/docs/cloud-messaging/android/topic-messaging